crowbar_framework/app/views/nodes/_buttons_power.html.haml

```diff
@@ -1,10 +1,23 @@
-- if @node.admin?
+.btn-group{ role: "group" }
   - if @node.bmc_set?
-    = link_to "Identify", hit_node_path(@node.handle, "identify"), :class => "btn btn-default", "data-remote" => true
-- else
-  = link_to "Reboot", hit_node_path(@node.handle, "reboot"), :class => "btn btn-default", "data-remote" => true, "data-confirm" => t("are_you_sure")
```

Comment:
Can you try to replace the attributes with ```class: "btn btn-default", remote: true, confirm: t("are_you_sure")```? That would simplify the view and should work on rails 4. The same some lines below.